sycl/source/detail/error_handling/enqueue_kernel.cpp

```diff
@@ -21,6 +21,54 @@ namespace detail {
 
 namespace enqueue_kernel_launch {
 
+bool l0HandleInvalidWorkGroupSize(const device_impl &DeviceImpl,
```

Comment:
Can't we reuse oclHandleInvalidWorkGroupSize instead?

Reply:
@romanovvlad, i am closing this PR as the issue was resolved by this https://github.com/intel/llvm/pull/2005.